plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -64,15 +64,24 @@ using boost::signals2::scoped_connection;
       wlog( "${details}", ("details",e.to_detail_string()) ); \
    }
 
-const fc::string logger_name("producer_plugin");
+const std::string logger_name("producer_plugin");
 fc::logger _log;
 
-const fc::string trx_successful_trace_logger_name("transaction_tracing");
+const std::string trx_successful_trace_logger_name("transaction_tracing");
```

Comment:
Go with the eosio/eos name of `transaction_success_tracing`. Make sure to update the example `logging.json`.